servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpServerConfig.java

```diff
@@ -128,6 +134,11 @@ public TcpServerConfig sslConfig(ServerSslConfig defaultSslConfig, Map<String, S
         return this;
     }
 
+    public TcpServerConfig sslListenMode(final SslListenMode mode) {
+        this.sslListenMode = mode;
```

Comment:
`requireNonNull(..)` Could also use some javadocs making it clear that this only has meaning if the `ServerSslConfig` is also set, otherwise it is ignored.